source/Symbiont.h

```diff
@@ -423,37 +445,76 @@ class Symbiont: public Organism {
     AddPoints(sym_portion);
     return host_portion * synergy;
   }
- 
- 
+
+
   /**
    * Input: None
-   * 
+   *
    * Output: The boolean representing if a symbiont
-   * will seek out to infect a host. 
-   * 
-   * Purpose: To determine if a symbiont wants to 
+   * will seek out to infect a host.
+   *
+   * Purpose: To determine if a symbiont wants to
    * infect a host based upon its infection chance
    */
   bool WantsToInfect(){
     bool result = random->GetDouble(0.0, 1.0) < infection_chance;
     return result;
   }
- 
- 
+
+
   /**
-   * Input: The size_t representing the location of the symbiont. 
-   * 
+   * Input: None
+   *
+   * Output: The boolean representing if a symbiont
+   * will survive crossing over into the host world.
+   *
+   * Purpose: To determine if a symbiont will survive
+   * crossing over into the host world based on infection risk.
+   */
+  bool InfectionFails(){
+    //note: this can be returned true, and an infecting sym can then be killed by a host that is already infected.
+    bool sym_dies = random->GetDouble(0.0, 1.0) < infection_failure_rate;
+    return sym_dies;
+  }
+
+  /**
+   * Input: The double representing the resources given by the world.
+   *
    * Output: None
-   * 
-   * Purpose: To process a symbiont, meaning to check for reproduction, distribute resources, 
-   * and to allow for movement 
+   *
+   * Purpose: Free living syms should be encouraged to keep the interaction values close to 0;
+   unused cooperation/antagonism without a host is wasteful, and so is punished.
+   This function copies the begining portion of Host's DistribResources.
+   */
+  void DistribResources(double resources){
+    double int_val = interaction_val;
+    if(my_host.IsNull()) { // this method should only be called on free-living syms, but double check!
+
+      if(int_val >= 0){
+	      double spent = resources * int_val;
+        this->AddPoints(resources - spent);
+      }
+      else {
+        double attack = -1.0 * int_val * resources;
+        this->AddPoints(resources - attack);
+      }
+    }
+  }
+
+  /**
+   * Input: The size_t representing the location of the symbiont.
+   *
+   * Output: None
+   *
+   * Purpose: To process a symbiont, meaning to check for reproduction, distribute resources,
+   * and to allow for movement
    */
   void Process(size_t location) {
     if (my_host.IsNull() && my_config->FREE_LIVING_SYMS()) { //free living symbiont
-      double resources = my_world->PullResources();
-      AddPoints(resources);
+      double resources = my_world->PullResources(my_config->FREE_SYM_RES_DISTRIBUTE()); //recieve resources from the world
+      DistribResources(resources); //penalize extreme sym vals if living outside a host
```

Comment:
I don't think we want this in line comment, and it will be more clear once the method is renamed